plugins/score-cycle-times.js

```diff
@@ -10,7 +10,7 @@ window.plugin.scoreCycleTimes = function() {};
 
 window.plugin.scoreCycleTimes.CHECKPOINT = 5*60*60; //5 hours per checkpoint
 window.plugin.scoreCycleTimes.CYCLE = 7*25*60*60; //7 25 hour 'days' per cycle
-
+window.plugin.scoreCycleTimes.localeTime = 'default';
```

Comment:
my point about user tastes, dev tastes and locales. Locales add complexity, illustrated by this small PR...
hour only is expected, we don't need minutes in case of cycle time... halfhour timezone... right I forgot about them